templates/vue-demo-store/components/product/ProductCard.vue

```diff
@@ -102,7 +102,6 @@ const srcPath = computed(() => {
       >
         <img
           ref="imageElement"
-          loading="lazy"
```

Comment:
Lazy was not needed here?

Reply:
as it's used in product listing, mostly, there is no need to have it loaded in a lazy way, due to possibly low CLS score.